projects/vanilla/src/app/search/search.component.ts

```diff
@@ -42,7 +43,14 @@ export class SearchComponent implements OnInit {
     "date": BsFacetDate
   }
 
-  metadata: string[] = [];
+  /**
+   * Returns the configuration of the metadata displayed in the facet-preview component.
+   * The configuration from the config.ts file can be overridden by configuration from
+   * the app configuration on the server
+   */
+  public get metadata(): MetadataConfig[] {
+    return this.appService.app?.data?.metadata as any as MetadataConfig[] || METADATA_CONFIG;
+  }
```

Comment:
Same comment here

Reply:
I will do this. But first, to fix de CI, I'll merge this changes